src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -23,24 +23,23 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds an order to the order book. "
             + "Parameters: "
+            + PREFIX_TID + "TRANSACTION_ID"
             + PREFIX_NAME + "NAME "
             + PREFIX_PHONE + "PHONE "
-            + PREFIX_EMAIL + "EMAIL "
             + PREFIX_ADDRESS + "ADDRESS "
             + PREFIX_DELIVERY_TIMESTAMP + "Delivery_DATE_&_TIME "
             + PREFIX_WAREHOUSE + "WAREHOUSE_LOCATION "
             + "[" + PREFIX_COMMENT + "COMMENT] "
-            + "[" + PREFIX_TAG + "TAG]...\n"
+            + "[" + PREFIX_TYPE + "TYPE_OF_ITEM] "
             + "Example: " + COMMAND_WORD + " "
+            + PREFIX_TID + "A999999 "
             + PREFIX_NAME + "John Doe "
             + PREFIX_PHONE + "98765432 "
-            + PREFIX_EMAIL + "johnd@example.com "
             + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
             + PREFIX_DELIVERY_TIMESTAMP + "2019-12-02 1500 "
             + PREFIX_WAREHOUSE + "5 Toh Guan Rd E, #02-30 S608831 "
             + PREFIX_COMMENT + "NIL "
-            + PREFIX_TAG + "friends "
-            + PREFIX_TAG + "owesMoney";
+            + PREFIX_TYPE + "glass";
```

Comment:
I did not see any cashOnDelivery in your message for add command. Please add inside.